packages/api/src/name.ts

```diff
@@ -95,6 +95,7 @@ export async function namePost (request: Request, env: Env, ctx: ExecutionContex
     if (error instanceof Error) {
       throw new HTTPError(`invalid ipns entry: ${error.message}`, 400)
     }
+    throw new HTTPError(`invalid ipns entry: ${error}`, 400)
```

Comment:
Just for consistency with other errors maybe we should capitalize Invalid?

Reply:
Thats a good point! - resolved